dbms/src/IO/VarInt.h

```diff
@@ -103,6 +103,13 @@ inline void readVarInt(Int16 & x, ReadBuffer & istr)
     x = tmp;
 }
 
+inline void readVarUInt(size_t & x, ReadBuffer & istr)
```

Comment:
Conflict when size_t and UInt64 are the same types.
Now it doesn't compile on Linux.

For proper fix, look at ReadHelpers.h, near line 705.